src/hits/hits.ts

```diff
@@ -35,30 +47,30 @@ import { NgAisInstantSearch } from '../instantsearch/instantsearch';
 export class NgAisHits extends BaseWidget {
   @ContentChild(TemplateRef) public template?: TemplateRef<any>;
 
-  // rendering options
-  @Input() transformItems?: Function;
+  @Input()
+  public transformItems?: <U extends HitsItem>(items: HitsItem[]) => U[];
 
-  // inner widget state returned from connector
-  public state: { hits: {}[]; results: {} } = { hits: [], results: {} };
+  public state: HitsState = {
+    hits: [],
+    results: {},
+  };
 
   constructor(
     @Inject(forwardRef(() => NgAisInstantSearch))
     public instantSearchParent: any
   ) {
     super('Hits');
-    this.createWidget(connectHitsWithInsights, { escapeHits: true });
+  }
+
+  ngOnInit() {
+    this.createWidget(connectHitsWithInsights, {
+      transformItems: this.transformItems,
+    });
```

Comment:
If we want to support `escapeHTML`, we should still pass the option.

Reply:
added in a stacked PR: https://github.com/algolia/angular-instantsearch/pull/578